docs/resources/samples/excel-cross-reference.md

```diff
@@ -123,28 +110,58 @@ function main(workbook: ExcelScript.Workbook, keys: string): string {
   }
+
   console.log("Returning: " + returnString);
   return returnString;
 }
 
 // An interface representing a row of event data.
 interface EventData {
-  event: string
+  eventId: string
   date: number
   location: string
   capacity: number
 }
 ```
 
-## Training video: Cross-reference and format an Excel file
+## Power Automate flow: Check for inconsistencies across the workbooks
+
+This flow extracts the event information from the first workbook and uses that data to validate the second workbook.
+
+1. Sign into [Power Automate](https://flow.microsoft.com) and create a new **Instant cloud flow**.
+1. Select **Manually trigger a flow** and press **Create**.
+1. Add a **New step** that uses the **Excel Online (Business)** connector with the **Run script** action. Use the following values for the action:
+    * **Location**: OneDrive for Business
+    * **Document Library**: OneDrive
+    * **File**: event-data.xlsx ([selected with the file chooser](../../testing/power-automate-troubleshooting.md#select-workbooks-with-the-file-browser-control))
+    * **Script**: Get event data
+
+    :::image type="content" source="../../images/cross-reference-flow-1.png" alt-text="The completed Excel Online (Business) connector for the first script in Power Automate.":::
+
+1. Add a second **New step** that uses the **Excel Online (Business)** connector with the **Run script** action. Use the following values for the action:
+    * **Location**: OneDrive for Business
+    * **Document Library**: OneDrive
+    * **File**: speaker-registration.xlsx ([selected with the file chooser](../../testing/power-automate-troubleshooting.md#select-workbooks-with-the-file-browser-control))
+    * **Script**: Validate speaker registration
+
+    :::image type="content" source="../../images/cross-reference-flow-2.png" alt-text="The completed Excel Online (Business) connector for the second script in Power Automate.":::
+1. This sample uses Outlook as the email client. You could use any email connector Power Automate supports. Add a **New step** that uses the **Office 365 Outlook** connector and the **Send and email (V2)** action. Use the following values for the action:
+    * **To**: Your test email account (or personal email)
+    * **Subject**: Event validation results
+    * **Body**: result (_dynamic content from **Run script 2**_)
 
-[Watch Sudhi Ramamurthy walk through this sample on YouTube](https://youtu.be/dVwqBf483qo").
+    :::image type="content" source="../../images/cross-reference-flow-3.png" alt-text="The completed Office 365 Outlook connector in Power Automate.":::
+1. Save the flow, then press **Test** to try it out. You should receive an email saying "Mismatch found. Data requires your review." This indicates there are differences between rows in **speaker-registrations.xlsx** and rows in **event-data.xlsx**. Open **speaker-registrations.xlsx** to see several highlighted cells where there are potential problems with the speaker registration listings.
```

Comment:
1. Favor "[select](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=26472)" over "[press](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=29040)" for inclusivity. 
2. When I tested this example, I had to click the "Save" button after entering the flow steps, and then go to "My flows" in the left hand pane, and then click the "Run" button next to this flow. Are you sure there's a Test button? I can't find it. 
```suggestion
1. Save the flow, then go to **My flows** and select **Run** next to this flow to try it out. You should receive an email saying "Mismatch found. Data requires your review." This indicates there are differences between rows in **speaker-registrations.xlsx** and rows in **event-data.xlsx**. Open **speaker-registrations.xlsx** to see several highlighted cells where there are potential problems with the speaker registration listings.
```

Reply:
Changed "press" and addressing the other comment as part of a subsequent PR.